app/controllers/api/users_controller.rb

```diff
@@ -1,17 +1,31 @@
 class API::UsersController < API::RestfulController
```

Comment:
Yeah I think that's a good call; do you mind if we make that change when we come to it though? It strikes me that other user profiles is a bit down the road from now (considering we don't really do it justice in beta)